docs/index.md

```diff
@@ -2,53 +2,113 @@
-    ```bash
-    sudo apt install python3-pip
+    ```powershell
+    [System.Environment]::SetEnvironmentVariable('PYENV',$env:USERPROFILE + "\.pyenv\pyenv-win\","User")
+
+    [System.Environment]::SetEnvironmentVariable('PYENV_ROOT',$env:USERPROFILE + "\.pyenv\pyenv-win\","User")
+
+    [System.Environment]::SetEnvironmentVariable('PYENV_HOME',$env:USERPROFILE + "\.pyenv\pyenv-win\","User")
+
+    [System.Environment]::SetEnvironmentVariable('path', $env:USERPROFILE + "\.pyenv\pyenv-win\bin;" + $env:USERPROFILE + "\.pyenv\pyenv-win\shims;" + [System.Environment]::GetEnvironmentVariable('path', "User"),"User")
     ```
 
-#### Verify Installation
+    Set execution policy to always be able to run Pyenv.
 
-To verify that you have Python installed, you can run the following command in your terminal:
+    ```powershell
+    Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy Unrestricted
+    ```
 
-=== "Windows"
+=== "Linux"
+    Open the terminal.
+
+    Install all required prerequisite dependencies.
 
     ```bash
-    py --version
+    sudo apt-get update; sudo apt-get install make build-essential libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev wget curl git llvm \
+    libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
     ```
-=== "Mac & Linux"
+
+    Download and execute installation script.
 
     ```bash
-    python3 --version
+    curl https://pyenv.run | bash
+    ```
+
+    <hr>
+    Optional steps but recommended.
+
+    Add the following entries into your `~/.bashrc` file.
+
+    ```sh
+    # pyenv
+    export PATH="$HOME/.pyenv/bin:$PATH"
+    eval "$(pyenv init --path)"
+    eval "$(pyenv virtualenv-init -)"
     ```
 
-!!! note "Note"
-    Depending on your Python installation, the version displayed may be different.
+    Restart shell and test.
+
+    ```bash
+    exec $SHELL
+    ```
 
-![Installation](assets/installation.png)
+=== "Mac"
 
-#### For Windows & Mac
+    Install Brew (Install apple devtools first if you have never used the terminal, it will prompt you automatically)
 
-When installing Python on Windows or Mac, you will need to make sure that you have the `Add Python to PATH` option checked. This will allow you to use Python in your command prompt or terminal.
+    ```zsh
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    ```
+
+    Installing Pyenv.
+
+    ```zsh
+    brew install pyenv
+    ```
+
+
+#### Installing Python
+
+To verify that you have Penv installed, you can run the following command in your terminal:
+
+```bash
+pyenv --version
+```
+
+Install Python and setting it globally.
+
+=== "Windows"
+
+    ```bash
+    pyenv install 3.10.5 && pyenv global 3.10.5 && python -V
+    ```
+=== "Mac & Linux"
+
+    ```bash
+    pyenv install 3.11 && pyenv global 3.11 && python -V
+    ```
```

Comment:
Why two different versions

Reply:
Because the pyenv from windows is always behind. The max version is 3.10.5.

Reply:
Okay